### DISCUSSION TITLE
Migration from v2 to v3 method

### DISCUSSION THREAD

**Discussion by n1nj444** [etcd-io/etcd#16841] [Q&A] [answered]
Hello,

I want to know if there's an equivalent to the method that was used in v2, for instance through this URL:

http://IP:PORT/v2/members

Because now when I try with v3, I get a "not found" error.

I checked this documentation https://etcd.io/docs/v3.3/dev-guide/api_reference_v3/#service-cluster-etcdserveretcdserverpbrpcproto, but I don't understand how to access the new URL for V3.

If you can assist, I'd appreciate it!

Thank you.

**Comment by chaochn47** [comment, marked as answer]
etcd v3.3 is out of supported window. 

> v3.5 is the current version, v3.4 is supported
https://etcd.io/docs/v3.6/op-guide/versioning/#service-versioning-and-supported-versions

For v2 to v3 migration, you can refer to https://etcd.io/docs/v3.4/op-guide/v2-migration/

Members API is exposed as part gRPC service in v3 now. Try accessing them with `etcdctl member list --endpoints http://IP:PORT`
